sdk/java-sdk-spring/src/it/java/com/example/wiring/actions/headers/EchoAction.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.wiring.actions.headers;
+
+import com.example.wiring.actions.echo.Message;
+import kalix.javasdk.action.Action;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+/**
+ * Action with the same in a different package.
```

Comment:
```suggestion
 * Action with the same name in a different package.
```

Or maybe something similar to `Action with the same name as com.example.wiring.actions.echo.ActionEcho` shouldn't raise `kalix.proxy.UserFunctionErrors$.unableToFindServiceInDescriptor`. I'm saying this to keep some context around for the future reader.